app/src/main/res/layout/activity_event_details.xml

```diff
@@ -1,171 +1,177 @@
-    </LinearLayout>
+            android:layout_marginLeft="10dp"
+            android:layout_marginRight="10dp"
+            android:orientation="horizontal"
+            android:paddingBottom="2dp"
+            android:paddingLeft="12dp"
+            android:paddingRight="12dp">
+
+            <TextView
+                android:id="@+id/tvTicketSold"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/tickets_amount_placeholder"
+                android:textAppearance="@style/TextAppearance.AppCompat.Small" />
+
+            <TextView
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/divide"
+                android:textAppearance="@style/TextAppearance.AppCompat.Small" />
+
+            <TextView
+                android:id="@+id/tvTicketTotal"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:text="@string/tickets_amount_placeholder"
+                android:textAppearance="@style/TextAppearance.AppCompat.Small" />
+        </LinearLayout>
+
+        <ProgressBar
+            android:id="@+id/progressTicketSold"
+            style="?android:attr/progressBarStyleHorizontal"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:layout_marginLeft="10dp"
+            android:layout_marginRight="10dp"
+            android:max="100"
+            android:maxHeight="10dip"
+            android:minHeight="10dip"
+            android:paddingLeft="12dp"
+            android:paddingRight="12dp"
+            android:progress="0" />
 
-    <TextView
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:text="Tickets Sold"
-        android:paddingBottom="0dp"
-        android:paddingLeft="12dp"
-        android:paddingRight="12dp"
-        android:paddingTop="12dp"
-        android:layout_margin="10dp"
-        android:textAppearance="@style/TextAppearance.AppCompat.Large"
-         />
-    <LinearLayout
-        android:layout_width="wrap_content"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:paddingBottom="2dp"
-        android:paddingLeft="12dp"
-        android:paddingRight="12dp"
-        android:layout_marginLeft="10dp"
-        android:layout_marginRight="10dp">
         <TextView
+            android:id="@+id/editText3"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="0"
-            android:textAppearance="@style/TextAppearance.AppCompat.Small"
-            android:id="@+id/tvTicketSold" />
+            android:layout_margin="10dp"
+            android:paddingBottom="0dp"
+            android:paddingLeft="12dp"
```

Comment:
@iamareebjamal use `R.dimen` for such values, so that it will be easy in future if it needs to changed . For example `R.dimen.padding_side`. Store dimension values in `res/`